leader-election-api/build.gradle

```diff
@@ -2,7 +2,7 @@ apply from: "../gradle/shared.gradle"
 
 libsDirName = file('build/artifacts')
 dependencies {
-  implementation project(":atlasdb-commons")
+  api project(":atlasdb-commons")
```

Comment:
Interesting. I'm guessing this is one where we actually expose something from commons in our public-facing part?

Reply:
Yeah, using the `implementation` configuration here gives us the following compilation error:

```
> Task :timelock-corruption-detection:timelock-corruption-detection-objects:compileJava FAILED
/Volumes/git/atlasdb/timelock-corruption-detection/timelock-corruption-detection-objects/src/generated/java/com/palantir/timelock/history/PaxosLogWithAcceptedAndLearnedValues.java:96: error: cannot access Persistable
                .paxosValue(Optional.of(paxosValue))
                ^
  class file for com.palantir.common.persist.Persistable not found
```

`PaxosValue` implements `Persistable` so `atlasdb-commons` needs to be on the compile classpath of `leader-election-api` consumers.